.gitignore

```diff
@@ -12,3 +12,6 @@ x64/
 
 # Doxygen documentation
 /doc/
+
+#Vcpkg project integration
+/packages/**
```

Comment:
Project typically includes a space between the `#` and the following text.

Generally newlines are included at the end of text files. (An exception can be made for machine generated text files, such as Visual Studio project files).